.github/workflows/reuseable-snapshot-timestamp.yml

```diff
@@ -196,15 +196,28 @@ jobs:
           git apply --verbose snapshot-timestamp/*
           rm -r snapshot-timestamp
       # Open pull request changes
-      - name: create pull request
+      - name: create pull request for no snapshot
+        if: ${{ inputs.disable_snapshot }}
+        uses: peter-evans/create-pull-request@153407881ec5c347639a548ade7d8ad1d6740e38 # v5.0.2
+        with:
+          commit-message: update timestamp
+          title: Update Timestamp
+          body: Sign timestamp file
+          branch: update-snapshot-timestamp
```

Comment:
any concern about this being the same branch as the one below?

Reply:
No, it was intentional due to https://github.com/sigstore/root-signing/blob/main/.github/workflows/scripts/review-pull-request.sh#L23.